### ISSUE TITLE
Undefined and null paths for empty and non-files.

### ISSUE THREAD

**Issue by nolanbconaway** [nolanbconaway/atom-reveal-in-finder#1]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.29.0 x64
**Electron**: 2.0.5
**OS**: Mac OS X 10.13.6
**Thrown From**: [reveal-in-finder](https://github.com/nolanbconaway/atom-reveal-in-finder) package 0.0.0

### Stack Trace

Uncaught TypeError: Cannot read property 'file' of undefined

```
At /Users/nconaway/Desktop/reveal-in-finder/lib/reveal-in-finder.js:26

TypeError: Cannot read property 'file' of undefined
    at Object.reveal (/Users/nconaway/Desktop/reveal-in-finder/lib/reveal-in-finder.js:26:60)
    at HTMLElement.revealInFinderReveal (/Users/nconaway/Desktop/reveal-in-finder/lib/reveal-in-finder.js:20:45)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:384:49)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:110:40)
```

### Commands

```
     -1:47.9.0 settings-view:open (input.hidden-input)
     -1:20.3.0 reveal-in-finder:reveal (div.package-detail.panels-item)
     -1:04.8.0 settings-view:open (a.btn-issue.btn.btn-error)
     -1:01.4.0 application:open-your-keymap (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  2x -0:57.5.0 core:backspace (input.hidden-input)
  2x -0:56 core:undo (input.hidden-input)
  2x -0:52.7.0 core:backspace (input.hidden-input)
     -0:52.1.0 core:save (input.hidden-input)
     -0:22.7.0 core:delete (input.hidden-input)
     -0:22.2.0 core:save (input.hidden-input)
     -0:21.4.0 core:close (input.hidden-input)
     -0:15.2.0 tree-view:remove-project-folder (span.name.icon.icon-repo)
```

**Comment by nolanbconaway** [comment]
Also handlers for when the file is not in focus but there is a project open.

**Comment by nolanbconaway** [comment]
fixed in v0.3.1
